go.mod

```diff
@@ -65,6 +65,8 @@ require (
 	github.com/marcboeker/go-duckdb v1.5.5
 	github.com/mazznoer/csscolorparser v0.1.3
 	github.com/mitchellh/mapstructure v1.5.0
+	github.com/pingcap/tidb v1.1.0-beta.0.20231124053542-069631e2ecfe
+	github.com/pingcap/tidb/pkg/parser v0.0.0-20231124053542-069631e2ecfe
```

Comment:
These are adding a lot of indirect deps. I wonder if removing the direct `tidb` import might reduce the indirect deps. It seems to only derive from this import:
```
	_ "github.com/pingcap/tidb/pkg/types/parser_driver"
```

Is there any way to avoid that?

Reply:
So there is a test driver mentioned in the quick start which is part of `pkg/parser` only. The doc says to use it for tests and use `tidb/parser_driver` for advanced use cases. I see all our test cases are passing when I also use test_driver so may be for now we can use it and in future if we find anything that doesn't work we can make a switch ?

Reply:
https://github.com/pingcap/tidb/blob/master/pkg/parser/docs/quickstart.md#parse-sql-text